webpack/components/TemplateSyncResult/components/SyncedTemplate/helpers.js

```diff
@@ -20,77 +50,85 @@ export const additionalInfo = (template, editPath) => {
     'templateFile',
   ];
 
-  return infoAttrs.map(attr => {
-    const key = itemIteratorId(template, attr);
-
+  return infoAttrs.map((attr, index) => {
     const classNameMap = { Ptable: 'Partition Table' };
 
     if (!template[attr]) {
-      return <EmptyInfoItem template={template} attr={attr} key={key} />;
+      return <ItemWrapper key={index}> </ItemWrapper>;
     }
 
     switch (attr) {
       case 'locked':
         return (
-          <IconInfoItem
-            template={template}
-            attr={attr}
-            iconName="lock"
-            tooltipText="Locked"
-            key={key}
-          />
+          <TooltipWrapper tooltipText="Locked" key={index}>
+            <ItemWrapper className="cell-info-padding">
+              <Icon>
+                <LockIcon />
+              </Icon>
+            </ItemWrapper>
+          </TooltipWrapper>
         );
       case 'snippet':
         return (
-          <IconInfoItem
-            template={template}
-            attr={attr}
-            iconName="check"
-            tooltipText="Snippet"
-            key={key}
-          />
+          <TooltipWrapper tooltipText="Snippet" key={index}>
+            <ItemWrapper className="cell-info-padding">
+              <Icon>
+                <CheckIcon />
+              </Icon>
+            </ItemWrapper>
+          </TooltipWrapper>
         );
       case 'humanizedClassName':
-        return (
-          <StringInfoItem
-            template={template}
-            attr={attr}
-            mapAttr={(templateObj, attribute) =>
-              classNameMap[templateObj[attribute]]
-                ? classNameMap[templateObj[attribute]]
-                : templateObj[attribute]
-            }
-            key={key}
-          />
-        );
       case 'kind':
-        return <StringInfoItem template={template} attr={attr} key={key} />;
       case 'templateFile':
         return (
-          <StringInfoItem template={template} attr={attr} key={key} elipsed />
+          <ItemWrapper key={index} className="cell-info-padding">
+            {extractAttr(template, attr, classNameMap)}
+          </ItemWrapper>
```

Comment:
This cancels the truncation and tooltip for the items, can you explain this change?